src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -245,19 +242,20 @@ uint16_t WebRTCTransportProviderServer::GenerateSessionId()
 // Command Handlers
 void WebRTCTransportProviderServer::HandleSolicitOffer(HandlerContext & ctx, const Commands::SolicitOffer::DecodableType & req)
 {
+    // TODO: this check needs to be removed after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11333
```

Comment:
No, this check is absolutely required by the conformance on the command fields.  It should be right here, but be responding with INVALID_COMMAND after that spec PR.